docs/architecture/incremental-append.md

```diff
@@ -111,14 +115,14 @@ Let's say the following data already exists into our data warehouse.
 ]
 ```
 
-In the next sync, the delta contains the following record:
+In the next sync, the delta applied to the source data contains the following record:
 ```javascript
 [
     { "name": "Louis XVI", "deceased": true, "updated_at":  1754 },
 ]
 ```
 
-At the end of this incremental sync, the data warehouse would still contain data from the first sync because the delta record did not provide a valid value for the cursor field (greater than last sync's max value, `1754 < 1755`).
+At the end of this incremental sync, the data warehouse would still contain data from the first sync because the delta record did not provide a valid value for the cursor field (the cursor field is not greater than last sync's max value, `1754 < 1755`), so it is not emitted by the source as a new or modified record.
```

Comment:
```suggestion
At the end of the second incremental sync, the data warehouse would still contain data from the first sync because the delta record did not provide a valid value for the cursor field (the cursor field is not greater than last sync's max value, `1754 < 1755`), so it is not emitted by the source as a new or modified record.
```